spec/api-web-frame-main-spec.ts

```diff
@@ -122,9 +122,59 @@ describe('webFrameMain module', () => {
     });
   });
 
+  describe('WebFrame.origin', () => {
+    it('should be null for a fresh WebContents', () => {
+      const w = new BrowserWindow({ show: false });
+      expect(w.webContents.mainFrame.origin).to.equal('null');
```

Comment:
The string null? That seems wrong

Reply:
Might be empty string after switching to `GURL::EmptyGURL()`?

Reply:
https://www.rfc-editor.org/rfc/rfc6454

![Screen Shot 2022-08-25 at 9 34 32 AM](https://user-images.githubusercontent.com/172800/186720853-9934c4b1-671e-448e-92e2-dae3160c616a.png)

Reply:
Can we add documentation that explicitly explains that this is a web origin string, directly include a note it could be the string `null` and then link to this doc?